presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraTokenSplitManager.java

```diff
@@ -43,20 +43,22 @@
 {
     private final CassandraSession session;
     private final int splitSize;
+    private final Optional<Long> configSplitsPerNode;
```

Comment:
How about using OptionalLong?

Reply:
Should I keep using `Optional<Long` as the above?

Reply:
Yes please.